docs/requirements.txt

```diff
@@ -17,6 +17,7 @@ Pygments==2.11.2
 pyparsing==3.0.7
 pytz==2021.3
 requests==2.27.1
+setuptools==69.0.3
```

Comment:
Is setuptools @ this version required for some features you make use of, or is this just the package version you happened to use?

Reply:
it's just the version I happened to use, it can be upgraded to 69.1.0, which I noticed was released the 12th, or it can be tested with lower versions.

Reply:
Thanks! Just wanted to check. We'll do the version testing later if needed.